spec/playload/parser/parser_spec.rb

```diff
@@ -78,16 +102,75 @@
     allow(create_covers).to receive(:call)
     allow(create_classifications).to receive(:call)
     allow(create_tech_specs).to receive(:call)
-    result
   end
 
   describe '#call' do
-    it 'calls all of the create helpers' do
-      expect(find_or_create_game).to have_received(:call).with(game_details: game_details)
+    context 'when creating a game' do
+      before do
+        result
+      end
+
+      it 'calls all of the create helpers' do
```

Comment:
`it 'calls all of the create helpers', :aggregate_failures do`

Optional: This will make failure reasons clearer if one or more of the assertions fail within an `it` block.